core/loader/utils.ts

```diff
@@ -25,13 +28,36 @@ function buildFullURL(link: HTMLLinkElement, baseUrl: string): string {
 }
 
 export function findLinks(text: TextResponse, type: string): string[] {
-  return [...text.parse().querySelectorAll('link')]
-    .filter(
-      link =>
-        link.getAttribute('type') === type &&
-        isString(link.getAttribute('href'))
-    )
-    .map(link => buildFullURL(link, text.url))
+  let links: string[] = []
+
+  links.push(
+    ...[...text.parse().querySelectorAll('link')]
+      .filter(
+        link =>
+          link.getAttribute('type') === type &&
+          isString(link.getAttribute('href'))
+      )
+      .map(link => buildFullURL(link, text.url))
+  )
+
+  links.push(
+    ...[...text.parse().querySelectorAll('a')]
+      .filter(a => {
+        let href = a.getAttribute('href')
+        if (!href) return false
+        return (
+          href.includes('feed.') ||
```

Comment:
Both Atom and RSS loader will find the same links which is not cool for performance.

Let’s pass regexp for `href` as argument to `findLinks` (like we do with `type`).

Reply:
Do you mean that checks on `feed.` and `feed.xml` is enough? I kept it only on `feed.` because it is valid for both cases, I also kept the check for `.atom` and `/atom` because I didn’t understand whether they should be removed or not)

I also added constants for href patterns for <a>, if this is a ok idea, maybe the type for <link> should also be added to the constants, and in findLinks just pass the key as an argument?